activemodel/lib/active_model/errors.rb

```diff
@@ -322,9 +322,13 @@ def add(attribute, message = :invalid, options = {})
     #  person.errors.added? :name, :too_long                                # => false
     #  person.errors.added? :name, "is too long"                            # => false
     def added?(attribute, message = :invalid, options = {})
-      message = message.call if message.respond_to?(:call)
-      message = normalize_message(attribute, message, options)
-      self[attribute].include? message
+      if message.is_a? Symbol
```

Comment:
I think we can remove this conditional and always read from the `details` hash.

Reply:
This would result in breaking changes, right? 
The `details` hash doesn't contain the message strings (normalised messages) and such a change wouldn't allow to call `added?` using the message string anymore.
I would expect, in my example test, that a call to `person.errors.added?(:name, 'is wrong')` is still possible and returns true.

Reply:
Ah yeah, in the case for asserting for the string correspondent to the symbol that would break.